scipy/optimize/_zeros_py.py

```diff
@@ -544,6 +544,22 @@ def bisect(f, a, b, args=(),
         Object containing information about the convergence. In particular,
         ``r.converged`` is True if the routine converged.
 
+    Notes
+    -----
+    As mentioned in the parameter documentation, the computed root ``x0`` will
+    satisfy ``np.isclose(x, x0, atol=xtol, rtol=rtol)``, where ``x`` is the
+    exact root. In equation form, this terminating condition is ``abs(x - x0)
+    <= xtol + rtol * abs(x)``.
```

Comment:
Technically,
```suggestion
    <= xtol + rtol * abs(x0)``.
```
right? That's how `isclose` is documented, and I am just guessing we could cook up a situation where the tolerances are loose and the difference could matter. (But it's possible that guess is wrong.)

Reply:
Yes, you're absolutely correct. I don't think what I wrote is even possible because the actual root isn't known so it's magnitude can't be used in the stopping criterion like that. I think there are situations where the difference could matter, I'll make a note to revisit that after I start getting ahead of my backlog.